packages/cli-v3/src/commands/deploy.ts

```diff
@@ -837,6 +840,8 @@ async function buildAndPushImage(
       `TRIGGER_CONTENT_HASH=${options.contentHash}`,
       "--build-arg",
       `TRIGGER_PROJECT_REF=${options.projectRef}`,
+      "--build-arg",
+      `NODE_EXTRA_CA_CERTS=${options.extraCACerts}`,
```

Comment:
What happens when this is an empty string or undefined?

Reply:
this variable by default is empty, nodejs only loads this variable if this has value during startup.

`When set, the well known "root" CAs (like VeriSign) will be extended with the extra certificates in file. The file should consist of one or more trusted certificates in PEM format. A message will be emitted (once) with [process.emitWarning()](https://nodejs.org/api/process.html#processemitwarningwarning-options) if the file is missing or malformed, but any errors are otherwise ignored.`

Reply:
converted this to check and apply arg instead of applying undefined.